config/initializers/telephony.rb

```diff
@@ -1,5 +1,7 @@
 Telephony.config do |c|
   c.adapter = Figaro.env.telephony_adapter.to_sym || :test
+  c.logger = Logger.new('log/telephony.log')
```

Comment:
Do we need to configure anything in devops to pick up this log?

Reply:
Should we set an explicit level on the logger so it doesn't log PII?

Reply:
The telephony logger is setup to make sure it doesn't log PII at any log level. It should never log PII at INFO which I think is the default, right?

Also, yes there's some devops work to picking these up in cloudwatch and logtrail

Reply:
Looks like default level is `DEBUG`
```
require 'logger'
Logger.new(STDOUT).level == Logger::DEBUG
# true
```

Reply:
Gotcha, I just updated it